### DISCUSSION TITLE
Managing tickLength by index

### DISCUSSION THREAD

**Discussion by maximerivolet** [airbnb/visx#1192] [Q&A] [answered]
Hi,

I would like to manage the tickLength on an axis. I would like odd values to have a certain tickLength, and even values another one. Can we split that ?

I tried to use two scale in parallel, but it didn't worked.

The goal is to have an alternation of labels, to avoid them to overlapse.

Do you have any solution to do that ?

Regards,

**Comment by VigneshTrantor** [comment, marked as answer]
@goldenivan is your problem related to https://github.com/airbnb/visx/discussions/1086 perhaps ?

**Reply by maximerivolet** [reply]
Hi,
It helped me for another problem.
To avoid trouble on this problem, I used the method explained here : https://github.com/airbnb/visx/pull/165
